webapp/app/.server/service/authorized-users.ts

```diff
@@ -9,7 +9,7 @@ const upgradeHandlerAddress = env.UPGRADE_HANDLER_ADDRESS;
 
 const range = (l: number): number[] => new Array(l).fill(0).map((_, i) => i);
 
-export const UserRole = z.enum(["guardian", "securityCouncil"]);
+export const UserRole = z.enum(["guardian", "securityCouncil", "anonymous"]);
```

Comment:
Added new role. Anonymous means that the user is not part of any multisig. Maybe there is a better word for this.